tools/Az.Tools.Predictor/README.md

```diff
@@ -4,19 +4,19 @@
 
 **[Az Predictor](https://www.powershellgallery.com/packages/Az.Tools.Predictor/)** is a PowerShell module that helps you navigate the cmdlets and parameters of the [Az module](https://www.powershellgallery.com/packages/Az) by providing context-aware suggestions in the PowerShell command line.
 
-Az Predictor is using the [subsystem plugin model](https://docs.microsoft.com/en-us/powershell/scripting/learn/experimental-features?view=powershell-7.1#pssubsystempluginmodel) available with PowerShell 7.1 and requires [PS Readline version 2.2.0](https://devblogs.microsoft.com/powershell/announcing-psreadline-2-1-with-predictive-intellisense/) to display the suggestions.
+Az Predictor is using the [subsystem plugin model](https://docs.microsoft.com/en-us/powershell/scripting/learn/experimental-features?view=powershell-7.1#pssubsystempluginmodel) available with PowerShell 7.1. This updated version requires [PS Readline 2.2 beta 2](https://devblogs.microsoft.com/powershell/announcing-psreadline-2-1-with-predictive-intellisense/) to display the suggestions.
```

Comment:
> requires [PS Readline 2.2 beta 2](https://devblogs.microsoft.com/powershell/announcing-psreadline-2-1-with-predictive-intellisense/) to display the suggestions

The link points to the PSReadLine 2.1 blog. You probably want to point to https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta2 instead.